ethcore/builtin/src/lib.rs

```diff
@@ -245,6 +245,13 @@ impl Builtin {
 	pub fn is_active(&self, at: u64) -> bool {
 		self.pricer.range(0..=at).last().is_some()
 	}
+
+	/// EIP 1352: Restricted address range for precompiles/system contracts
+	#[inline]
```

Comment:
```suggestion
```
unless you have a benchmark that proves it to be useful, compile should know better